koku/masu/processor/expired_data_remover.py

```diff
@@ -130,18 +130,13 @@ def remove(self, simulate=False, provider_uuid=None, line_items_only=False):
             ([{}]) List of dictionaries containing 'account_payer_id' and 'billing_period_start'
 
         """
-        removed_data = None
+        removed_data = []
         disable_purge_line_item = (Provider.PROVIDER_AZURE, Provider.PROVIDER_AZURE_LOCAL, Provider.PROVIDER_OCP)
```

Comment:
I think we can remove this since line item data removal is now gone.

Reply:
There are still data returned as a list even though line-item data are not deleted. This is for account info.